hack/local-up-cluster.sh

```diff
@@ -714,6 +717,15 @@ function start_kubedns {
     fi
 }
 
+function start_kubedashboard {
+    if [[ "${ENABLE_CLUSTER_DASHBOARD}" = true ]]; then
+        echo "Creating kubernetes-dashboard"       
+        # use kubectl to create the dashboard
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
```

Comment:
We should think really hard before just installing with random manifests from the internet.  This is the moral equivalent of `curl | sudo bash` for kubernetes as we are using admin creds.  Are there no released or local versions that we can point to?